File: base64/_Nim/b64.nim
import base64, nimpy

{.passL: "-fPIC -shared".}

proc b64e(s: string): string {.exportpy.} = 
  result = base64.encode(s)

proc b64d(s: string): string {.exportpy.} =
  result = base64.decode(s)

when defined(release):
  assert b64d("dGVzdA==") == "test"
  assert b64e("test") == "dGVzdA=="

File: base64/_Nuitka/b64.py
import base64

def b64e(s):
  return base64.b64encode(str(s).encode())

def b64d(s):
  return base64.b64decode(s).decode()

if __name__ == "__main__":
  assert b64e("test") == b"dGVzdA=="
  assert b64d("dGVzdA==") == "test"